<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="L_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Buku</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Judul</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_14">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pengarang</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_15">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Buku</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_16">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Penerbit</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_18">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Jumlah Eksemplar</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_17">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Jumlah Terpinjam</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s1_Buku_ID">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="w1Judul">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w1Pengarang">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w1Penerbit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w1Eksemplar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w1Terpinjam">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b1_Buku_ID">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s2_Member_ID">
    <property name="geometry">
     <rect>
      <x>542</x>
      <y>100</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b2_Member_ID">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Peminjam</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Member</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nama</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_5">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Maks Meminjam</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Telah Meminjam</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b2_Confirm">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Konfirmasi Peminjaman</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_7">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Transaksi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="s3_Transaksi_ID">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b3_Transaksi_ID">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>411</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Judul Buku</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>441</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ID Peminjam</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>377</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanggal Meminjam</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_13">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>411</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanggal Kembali</string>
    </property>
   </widget>
   <widget class="QLabel" name="w3Peminjam">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w3Kembali">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w3Tgl_Pinjam">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>380</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w3Judul">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w2Nama">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w2Max_Pinjam">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="w2Total_Pinjam">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b3_Update">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b3_Delete">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="L_19">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QComboBox" name="w3Status">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dipinjam</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Telah Dikembalikan</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="b_Back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <zorder>s2_Member_ID</zorder>
   <zorder>L_2</zorder>
   <zorder>L_10</zorder>
   <zorder>L_14</zorder>
   <zorder>L_15</zorder>
   <zorder>L_16</zorder>
   <zorder>L_18</zorder>
   <zorder>L_17</zorder>
   <zorder>s1_Buku_ID</zorder>
   <zorder>w1Judul</zorder>
   <zorder>w1Pengarang</zorder>
   <zorder>w1Penerbit</zorder>
   <zorder>w1Eksemplar</zorder>
   <zorder>w1Terpinjam</zorder>
   <zorder>b1_Buku_ID</zorder>
   <zorder>b2_Member_ID</zorder>
   <zorder>L</zorder>
   <zorder>L_3</zorder>
   <zorder>L_4</zorder>
   <zorder>L_5</zorder>
   <zorder>L_6</zorder>
   <zorder>b2_Confirm</zorder>
   <zorder>L_7</zorder>
   <zorder>L_8</zorder>
   <zorder>s3_Transaksi_ID</zorder>
   <zorder>b3_Transaksi_ID</zorder>
   <zorder>L_9</zorder>
   <zorder>L_11</zorder>
   <zorder>L_12</zorder>
   <zorder>L_13</zorder>
   <zorder>w3Peminjam</zorder>
   <zorder>w3Kembali</zorder>
   <zorder>w3Tgl_Pinjam</zorder>
   <zorder>w3Judul</zorder>
   <zorder>w2Nama</zorder>
   <zorder>w2Max_Pinjam</zorder>
   <zorder>w2Total_Pinjam</zorder>
   <zorder>b3_Update</zorder>
   <zorder>b3_Delete</zorder>
   <zorder>L_19</zorder>
   <zorder>w3Status</zorder>
   <zorder>b_Back</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
